### ISSUE TITLE
A11y docs: Consider changing text sizes from px to rem/em

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1344]
CENG_3419

### Description:
Paragraph font-sizes (and font-sizes across the system) are set in `px`. While full-page zoom (pinch zoom on touch devices) is often seen as sufficient for text resizing, setting font-sizes in `px` means users cannot resize the text in browser or OS preferences.

### Remediation:
Switching to using `rem` (and `em` where applicable) for font-sizes is recommended to meet WCAG 1.4.4.
